src/main/java/org/folio/event/handler/HoldingUpdateAsyncRecordHandler.java

```diff
@@ -76,7 +81,7 @@ private Future<List<PoLine>> processPoLinesUpdate(HoldingEventHolder holder, Con
       })
       .compose(poLines -> updatePoLines(holder, poLines, conn))
       .compose(poLines -> updateTitles(holder, poLines, conn).map(poLines))
-      .compose(poLines -> auditOutboxService.saveOrderLinesOutboxLogs(conn, poLines, OrderLineAuditEvent.Action.EDIT, holder.getHeaders()).map(poLines));
+      .onSuccess(poLines -> auditOutboxService.saveOrderLinesOutboxLogs(conn, poLines, OrderLineAuditEvent.Action.EDIT, holder.getHeaders()).map(poLines));
```

Comment:
can we remain compose here?

Reply:
> can we remain compose here?

You mean to have 2 compose in a TX for PoLine and Piece, then followed by that 1 for PoLine and 1 for Piece outbox, 4 compose commands chained?

Reply:
> can we remain compose here?

I see what you mean, change onSuccess to compose, yeah it can be done

Reply:
yes, 4 compose like this
![image](https://github.com/user-attachments/assets/b6956d00-2e50-4939-9eaa-a0de57e86c70)

Reply:
because if we remain this logic in onSuccess we will have success future as result so will not know about failure if could not write to audit log